src/ms.tex

```diff
@@ -266,6 +266,46 @@ \subsection{Affiliated packages}
 Include a big table of all affiliated packages and references (as in v2.0
 paper).
 
+All affiliated packages are listed on the astropy 
+website\footnote{\url{https://www.astropy.org/affiliated}}. Since 
+\cite{astropy:2018}, there has been an explosion of gravitational-wave affiliated 
+packages including: \texttt{PyCBC} for exploring GW sources, lenstronomy for 
+modeling strong gravitational lenses, \texttt{ligo.skymap} for visualizing GW 
+probability maps, and \texttt{EinsteinPy} for general relativity and gravitational astronomy. 
+There have been several packages related to HEALPix: \texttt{astropy-healpix}
+for a BSD licensed HEALPix implementation, and \texttt{mocpy} for Multi-Order 
+Coverage maps. \texttt{astroalign} as been introduced for astrometric registration, 
+and \texttt{python-cpl} has been added for ESO pipelines and VLT data products. 
+For ground-based astronomy, \texttt{baseband} has added IO capabilities for 
+VLBI, and \texttt{SpectraPy} brings slit spectroscopy to the astropy ecosystem. 
+We have \texttt{agnpy} for AGN astrophysics, and \texttt{statmorph} for fitting galactic 
+morphological diagnostics. \texttt{saba} gives an interface to sherpa's fitting 
+routines. For modeling interstellar dust extinction we have 
+\texttt{dust_extinction}, and for extracting features from time series we 
+have \texttt{feets}. We also have \texttt{corral} for managing data intensive parallel 
+pipelines. \texttt{BayesicFitting} provides an interface for generic Bayesian 
+inference, and \texttt{sbpy} does calculations for asteroid and cometary 
+astrophysics. Finally, \texttt{synphot} provides an interface for synthetic photometry.
+
+Several of the affiliated packages described in [reference to paper 2] have
```

Comment:
I don't know if it's relevant, but `stingray` has had a significant re-write and performance overhaul, plus better interoperability with other code (e.g. `EventList` and `Lightcurve` classes round-tripping to Astropy's `Timeseries`/`Table` and to `lightkurve.LightCurve` objects).
Also, `HENDRICS` has new fast pulsar search methods implemented. The next release of both packages should be before this paper's submission.

Reply:
Got it, thanks!